src/main/flight/pid.c

```diff
@@ -105,9 +105,10 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .pidSumLimitYaw = PIDSUM_LIMIT_YAW,
         .yaw_lpf_hz = 0,
         .dterm_lpf_hz = 100,    // filtering ON by default
+        .dterm_lpf_2_hz = 200,    // second PT1 ON by default
```

Comment:
let's default to `0` (off) for now.

Reply:
OK

Reply:
We are not at release candidate stage presently but I would support/entertain a PR setting more aggressive defaults leading up to (or during) that phase. The general guidance I have received is to implement the functionality (default to off).

Reply:
no problem